packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -204,4 +204,94 @@ void main() {
     expect(tester.getRect(find.byType(AnimatedSize).at(1)), new Rect.fromLTWH(0.0, 56.0 + 1.0 + 56.0, 800.0, 0.0));
     expect(tester.getRect(find.byType(AnimatedSize).at(2)), new Rect.fromLTWH(0.0, 56.0 + 1.0 + 56.0 + 16.0 + 16.0 + 48.0 + 16.0, 800.0, 100.0));
   });
+
+  testWidgets('Single Panel Open Test',  (WidgetTester tester) async {
```

Comment:
Make sure you test adding/removing panels in radio mode (especially the open panel), and switch from radio to non radio